index.js

```diff
@@ -51,7 +52,7 @@ function handlebars(data, opts) {
 		if (depth > maxDepth) { return; }
 		base = base || dir;
 		fs.readdirSync(dir).forEach(function (basename) {
-			var filename = dir + '/' + basename;
+			var filename = path.join(dir, basename);
```

Comment:
We need to add
``` 
dir = path.normalize(dir);
```
before execute

```
registerPartials(dir, dir, 0);
```

Example
```
	// Go through a partials directory array
	if(options.batch){
		// Allow single string
		if(typeof options.batch === 'string') options.batch = [options.batch];

		options.batch.forEach(function (dir) {
			dir = path.normalize(dir);
			registerPartials(dir, dir, 0);
		});
	}
```

If it willn't be fixed, we will have normalized "filename" and not normalized "basename" variables;